Makefile

```diff
@@ -16,17 +19,25 @@ PREFER_LOWEST ?=
 # Building
 
 build: ## Install the dependencies
-	make 'composer-install --optimize-autoloader --ignore-platform-reqs'
+	make 'composer-install --optimize-autoloader'
 
 build-update: ## Update the dependencies
-	make 'composer-update --optimize-autoloader --ignore-platform-reqs ${PREFER_LOWEST}'
+	make 'composer-update --optimize-autoloader ${PREFER_LOWEST}'
+
+ensure-composer-file: # Update the composer file
+ifeq (${OS},Darwin)
+	sed -E -e 's/"php": "[0-9\.]+"/"php": "${PHP_VER}"/' -i '' composer.json
```

Comment:
As discussed in slack, this editing is a bit icky but I understand why it was done.